Wrappers/Python/test/test_algorithms.py

```diff
@@ -300,7 +300,7 @@ def setup(data, dnoise):
             return noisy_data, alpha, g
 
         noisy_data, alpha, g = setup(data, dnoise)
-        operator = GradientOperator(ig, correlation=GradientOperator.CORRELATION_SPACE)
+        operator = GradientOperator(ig, correlation=GradientOperator.CORRELATION_SPACE, backend='numpy')
```

Comment:
why changing to the non default option?

Reply:
the C backend has fewer combinations. before it was silently defaulting to numpy, not it raises a warning. The warnings were spamming the output. So the test hasn't changes, it was running the non-default one anyway. But now we know that's the case if we need to debug.